src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -33,6 +33,14 @@ import './p5.RenderBuffer';
 p5.RendererGL.prototype.beginShape = function(mode) {
   this.immediateMode.shapeMode =
     mode !== undefined ? mode : constants.TESS;
+  if (this._useUserAttributes === true){
+    for (const name of Object.keys(this.userAttributes)){
```

Comment:
Is this equivalent to this? Might save making an intermediate array
```suggestion
    for (const name in this.userAttributes){
```

Reply:
> Is this equivalent to this? Might save making an intermediate array

Thanks, I've replaced all references!